README.md

```diff
@@ -147,6 +147,25 @@ archive jobs initiated.  This is the one part of the interactive tool which is
 stateful.  There is no permanent record of these executed command lines, so if
 you start a new interactive plotman session, this log is empty.
 
+## `plotman` commands
```

Comment:
Don't move it now, but this would generally be good as a separate PR since it is mostly unrelated.  I can see a relationship to the passive mode being added to interactive, though it isn't pointed out here.  So maybe that's the real comment, to note that along with running plot as a daemon you can then set interactive to not autostart plotting.